src/reducers/app.js

```diff
@@ -183,6 +184,31 @@ const isNewlyPublished: Reducer<boolean> = (state = false, action) => {
   }
 };
 
+const isDragAndDropDragging: Reducer<boolean> = (state = false, action) => {
```

Comment:
Reducers are also a good place for some light code documentation. See the reducer above as an example.